### DISCUSSION TITLE
I double-clicked connect, but it didn't work.

### DISCUSSION THREAD

**Discussion by ClairePark1130** [Serial-Studio/Serial-Studio#178] [Q&A] [answered]
Hi, 

I connected my Arduino board and pull serial studio up to get a reading of force sensor's data. 
But, it didn't work... When double-clicking "connect", it changes "disconnect" but came back to "connect"... 

What happened???
<img width="1119" alt="Screenshot 2024-03-10 at 8 34 29 PM" src="https://github.com/Serial-Studio/Serial-Studio/assets/162936429/f6dda9c0-d5ba-4403-a4da-219b2b497d60">

**Comment by alex-spataru** [comment, marked as answer]
Please refer to the [wiki](https://github.com/Serial-Studio/Serial-Studio/wiki/Getting-Started#connecting-a-device-quick-plot-mode) for an up-to-date tutorial on using Serial Studio, as well as an overview of the different [operation modes](https://github.com/Serial-Studio/Serial-Studio/wiki/Operation-Modes).

Starting from version 3.0.0, Serial Studio includes a 'Quick Plot Mode,' which requires no project setup and uses a format similar to the Arduino Plotter. However, this mode is limited to displaying data in a multiplot, a datagrid, and individual plots for each channel. If you need more advanced configuration options or wish to use widgets, you'll need to set up a project using the [Project Editor](https://github.com/Serial-Studio/Serial-Studio/wiki/Project-Editor)."
